app/controllers/admin/domain_allows_controller.rb

```diff
@@ -25,6 +25,7 @@ def create
   def destroy
     authorize @domain_allow, :destroy?
     UnallowDomainService.new.call(@domain_allow)
+    log_action :destroy, @domain_allow
```

Comment:
This was apparently missing, I only noticed it when doing an audit on all the log_action calls to make sure I'd found everything that referred to a domain/instance.

Reply:
Extracted to https://github.com/mastodon/mastodon/pull/30125